src/App.jsx

```diff
@@ -14,70 +15,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [
+    currentValue, setCurrentValue,
+  ] = useState('Jam');
```

Comment:
Please provide more representive state name, for example selectedGood, setSelectedGood
```suggestion
export const App = () => {
  const [currentValue, setCurrentValue] = useState('Jam');
```